kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -121,26 +121,19 @@ define([
             // match app_id to get the abbreviated name for the analysis
             if (typesToFiles[type].appId === jobInfo.app_id) {
                 analysisType = fileTypeMapping[type];
-                Object.keys(jobInfo.job_params[0]).forEach((param) => {
-                    // find each parameter in job_params in the app specs
-                    appData.specs[jobInfo.app_id].parameters.forEach((appParam) => {
-                        // only add values that are in the parameters
-                        // as these are the user-settable params
-                        if (appParam.id === param) {
-                            const value = jobInfo.job_params[0][param];
-                            const label = appData.specs[jobInfo.app_id].parameters
-                                .filter((p) => {
-                                    return param === p.id;
-                                })
-                                .map((p) => {
-                                    return p.ui_name;
-                                })[0];
-                            params[param] = {
-                                value,
-                                label,
-                            };
-                        }
-                    });
+                Object.values(appData.specs[jobInfo.app_id].parameters).forEach((param) => {
+                    const paramId = param.id;
+                    // params generated by the job manager
+                    if (jobInfo.userParams && jobInfo.userParams[paramId]) {
+                        const value = jobInfo.userParams[paramId];
+                        const label = param.ui_name;
+                        params[paramId] = { value, label };
+                        // params from a job info call
+                    } else if (jobInfo.job_params && jobInfo.job_params[0][paramId]) {
```

Comment:
this is a nuisance -- we don't want to have two different formats for the job info, but there are already existing narratives with info in this format. I will unify the format in another PR.